__generator__/builtin.yml

```diff
@@ -647,8 +647,8 @@ std.collect:
   reference: "https://developer.fastly.com/reference/vcl/functions/miscellaneous/std-collect/"
   on: [RECV, HASH, HIT, MISS, PASS, FETCH, ERROR, DELIVER, LOG]
   arguments:
-    - [ID]
-    - [ID, STRING]
+    - [STRING]
+    - [STRING, STRING]
```

Comment:
This is so tricky! 
Falco treats any `req.%any.%any` as a string and that makes sense because we are evaluating them eventually!
However, `std.collect` treats them as Identifier for some reason. 
I didn't really invest much time to handle this case! 
I would love to hear more from you here and how you would treat this case!